charts/frontend/templates/prometheusrules.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: frontend.rules
+  namespace: monitoring
+spec:
+  groups:
+  - name: frontend
+    rules:
+    - alert: FrontendNotDeployedSuccessfully
+      annotations:
+        description: "Frontend has not deployed successfully in namespace {{ "{{" }} $labels.namespace }}"
```

Comment:
Might want to say that this is just an example alert to demonstrate PrometheusRule. Otherwise people might get the wrong idea - a lot of our users won't have any experience with designing alerts so it could be confusing.